src/queue-list/laboratory-tabs.component.tsx

```diff
@@ -36,6 +48,18 @@ const LaboratoryQueueTabs: React.FC = () => {
             <Tab>{t("referredTests", "Referred tests")}</Tab>
             <Tab>{t("reviewList", "Review")}</Tab>
             <Tab>{t("approveList", "Approved")}</Tab>
+            {extensions.map((extension, index) => (
```

Comment:
Add a check for existence of extensions so that a blank panel isn't rendered. Also i think we need a better name. Extensions seems very generic.